app/services/bulk_action_persister.rb

```diff
@@ -55,7 +55,15 @@ def job_params
       set_governing_apo: set_governing_apo,
       manage_catkeys: manage_catkeys,
       prepare: prepare,
+      create_virtual_objects: csv_string(create_virtual_objects),
       groups: groups
     }
   end
+
+  def csv_string(params)
```

Comment:
can this be named so its unique purpose is clearer?   Maybe csv_file?  Or, if it's only for create_virtual_objects, indicate THAT in the method name?  and ... you do a File.read here?  Not a File.exists?  I'm not up on my bulk actions, so maybe it makes sense ...

Reply:
It returns a CSV in string form. I can reflect that it's used for virtual objects in the method name. In this case, the file being read is the file that was just uploaded by the user.